packages/template/init.lua

```diff
@@ -0,0 +1,21 @@
+local base = require("packages.base")
+
+local package = pl.class(base)
+package._name = "template"
+
+function package:_init()
+  base._init(self)
+end
+
+function package:registerCommands()
+  self:registerCommand("package-command", function(_, _)
+  end)
```

Comment:
You could minimally do a `SILE.process(content)` in these example commands, just processing the content without change.